canonical-web-and-design/microk8s.io
# Move browser test to issue template

## Done

Move browser test to issue template

## QA

- Check out this feature branch
- See that the request for browser details is in the issue template, not the PR one
